tsdb/engine/tsm1/engine.go

```diff
@@ -1364,13 +1365,15 @@ func (e *Engine) deleteSeriesRange(seriesKeys [][]byte, min, max int64) error {
 	if len(seriesKeys) > 0 {
 		buf := make([]byte, 1024) // For use when accessing series file.
 		ids := tsdb.NewSeriesIDSet()
+		measurements := make(map[string]struct{}, 1)
+
 		for _, k := range seriesKeys {
 			if len(k) == 0 {
 				continue // This key was wiped because it shouldn't be removed from index.
 			}
 
-			name, tags := models.ParseKey(k)
-			sid := e.sfile.SeriesID([]byte(name), tags, buf)
+			name, tags := models.ParseKeyBytes(k)
+			sid := e.sfile.SeriesID(name, tags, buf)
```

Comment:
A few lines down is:

```go
if k == nil {
    continue
}
```

That's not longer possible with the `emptyBytes` addition, and so can be removed.